CodeAcross/index.html

```diff
@@ -168,10 +162,10 @@ <h2 class="date">February 21 2015</h2>
 
             <div class="border">
                 <div class="gallery">
-                    <img src="../img/CfA_logo_lg.png" width="250" />
-                    <img src="../img/rally-logo.png" width="250" />
-                    <img src="../img/galvanize.png" width="250"/>
-                    <img src="../img/you-here.png" width="250" />
+                    <a href="http://www.codeforamerica.org/"><img src="../img/CfA_logo_lg.png" width="250" /></a>
+                    <a href="https://www.rallydev.com/"><img src="../img/rally-logo.png" width="250" /></a>
+                    <a href="http://www.galvanize.it/"><img src="../img/galvanize.png" width="250"/></a>
+                    <a href="#"><img src="../img/you-here.png" width="250" /></a>
```

Comment:
Oh yeah I like the idea for the actual logos. I was just referring to the last image here.